src/Sylius/Bundle/CoreBundle/Resources/config/state-machine.yml

```diff
@@ -36,6 +32,12 @@ winzou_state_machine:
             complete:
                 from: [payment_selected]
                 to: completed
+        callbacks:
+            after:
+                recalculate_order:
+                    on: ['address', 'readdress', 'select_shipping', 'reselect_shipping', 'select_payment', 'reselect_payment', 'complete']
```

Comment:
Order should not be recalculated on ``complete``. This happens when you confirm your order, so we should leave it as is, especially we display it to the customer on checkout summary.